_rules/zoom-text-no-overflow-clipping-59br37.md

```diff
@@ -0,0 +1,247 @@
+
+- **line wrapping**: A [computed][] [line-height][] equal to or greater than its
+  height of its [bounding box][], or in case of a [computed][] [overflow][] of `clip`, its [content box][].
+
+## Assumptions
+
+If any of the following assumptions is true, failing this rule may not result in a failure of [success criterion 1.4.4 Resize text](https://www.w3.org/TR/WCAG21/#resize-text):
+
+- There is no other mechanism for resizing text available on the page, that can be used to resize text to 200% without loss of information or functionality. This includes font resizing in the browser, or a javascript mechanism of resizing in the page.
+
+- [Text nodes][text node] can not be [clipped by overflow][clipped] without loss of information, except for [text nodes][text node] with an [ancestor][] with `aria-hidden` set to `true`, or when specific styles have been applied to ensure text is clipped cleanly (text-overflow, line wrapping or hidden text).
+
+- While [success criterion 1.4.4 Resize text](https://www.w3.org/TR/WCAG21/#resize-text) does not explicitly mention which display size has to be resized up to 200%, it is assumed that a [display size][] of 1280 by 1024 is applicable. A 1280 by 1024 [display size][] is explicitly mentioned under [success criterion 1.4.10 Reflow](https://www.w3.org/TR/WCAG21/#reflow).
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.4: Resize text](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html)
+- [G42: Using a technology that has commonly-available user agents that support zoom](https://www.w3.org/WAI/WCAG21/Techniques/general/G142)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This [text node][] is fully [visible][] at a [display size][] of 640 by 512.
+
+```html
+<div style="white-space: nowrap; overflow: hidden;">
+	Once upon a midnight dreary, while I pondered, weak and weary,<br />
+	Over many a quaint and curious volume of forgotten lore.<br />
+	While I nodded, nearly napping, suddenly there came a tapping,<br />
+	As of some one gently rapping, rapping at my chamber door.<br />
+	“’Tis some visitor,” I muttered, “tapping at my chamber door.<br />
+	Only this and nothing more.”
+</div>
+```
+
+#### Passed Example 2
+
+This [text node][] is [clipped][] using `text-overflow: ellipsis` at a [display size][] of 640 by 512. A link to a full version of the poem is also provided.
+
+```html
+<div style="white-space: nowrap; overflow: hidden; text-overflow: ellipsis;">
+	Once upon a midnight dreary, while I pondered, weak and weary, Over many a quaint and curious volume of forgotten
+	lore. While I nodded, nearly napping, suddenly there came a tapping.
+</div>
+<a href="/test-assets/59br37/poem.html">Full text of the poem</a>
+```
+
+#### Passed Example 3
+
+This [text node][] is [clipped][] using `text-overflow: ellipsis` at a [display size][] of 640 by 512. A link to a full version of the poem is also provided.
```

Comment:
This description is duplicate of Passed example 2 and `text-overflow: ellipsis` is not presented in the  example code.